msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -380,6 +383,26 @@ protected PublicClientApplication(@NonNull final Context context,
         }
     }
 
+    private void setupTelemetry(@NonNull final Context context,
+                                @Nullable final PublicClientApplicationConfiguration developerConfig) {
+        if (null != developerConfig.getTelemetryConfiguration()) {
+            com.microsoft.identity.common.internal.logging.Logger.verbose(
+                    TAG,
+                    "Telemetry configuration is set. Telemetry is enabled."
+            );
+        } else {
+            com.microsoft.identity.common.internal.logging.Logger.verbose(
```

Comment:
what would happen if we're calling the constructor above with configuration as null?

Reply:
If the telemetry configuration is null, then the Telemetry.isTelemetryEnabled will be set to "false", meaning that the Telemetry is disabled for this client.